tests/component/test_watchdog_properties.py

```diff
@@ -0,0 +1,171 @@
+"""Tests for validating watchdog properties."""
+
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx.constants import Level, Edge
+from nidaqmx.errors import DaqError, DAQmxErrors
+from nidaqmx.system.watchdog import DOExpirationState
+
+
+def test__watchdog__get_expired__returns_false(any_x_series_device):
```

Comment:
"Given watchdog" is close, but "given watchdog task" is clearer. Rename all tests to `test__watchdog_task__...`